buildspecs/linux_ppc-64_cmprssptrs_le_valhalla_nestmates.spec

```diff
@@ -0,0 +1,283 @@
+		<property name="graph_req.aux0" value="{$common.req.build.cuda$}"/>
+		<property name="graph_req.aux1" value=""/>
+		<property name="graph_req.build" value="{$common.req.build.java9$}"/>
+		<property name="graph_req.build2" value="{$common.req.build.java8$}"/>
+		<property name="graph_req.machine" value="{$machine_mapping.ppc64le$}"/>
+		<property name="graph_req.machine.test" value="{$spec.property.graph_req.machine$}"/>
+		<property name="graph_req.os" value="{$machine_mapping.linux$}"/>
+		<property name="graph_req.os.build" value="{$spec.property.graph_req.os$}"/>
+		<property name="graph_req.os.perf" value="{$spec.property.graph_req.os$}"/>
+		<property name="graph_se_classlib.java5" value="jcl_se.zip"/>
+		<property name="graph_se_classlib.java6" value="jcl_se.zip"/>
+		<property name="graph_variant.testing_suffix" value=""/>
+		<property name="graph_variant.trailingID" value=""/>
+		<property name="isReallyUnix" value="true"/>
+		<property name="j2seRuntimeDir" value="jre/lib/ppc64le"/>
+		<property name="j2seTags" value="pxp64lecmprssptrs60,j9vmxp64lecmprssptrs24"/>
+		<property name="j9dt.compileTarget" value="makefile"/>
+		<property name="j9dt.make" value="gmake"/>
+		<property name="j9dt.toolsTarget" value="buildtools.mk"/>
+		<property name="javatestPlatform" value="linux_ppc-64_le_cr"/>
+		<property name="jclMaxSecurityPolicyDefine" value=" &quot;-Djava.security.policy=http://jcl1.ottawa.ibm.com/testres/java.policy&quot;"/>
+		<property name="jclMemoryMax" value="-Xmx64m"/>
+		<property name="jclOSStackSizeMax" value=""/>
+		<property name="jgrinderTestingSupported" value="true"/>
+		<property name="jitTestingOptLevel" value="optlevel=warm"/>
+		<property name="localRootPath" value="$(J9_UNIX_ROOT)"/>
+		<property name="longLimitCmd" value=""/>
+		<property name="main_shortname" value="xl64"/>
+		<property name="os.lineDelimiter" value="unix"/>
+		<property name="platform_arch" value="ppc64le"/>
+		<property name="svn_stream" value=""/>
+		<property name="uma_make_cmd_ar" value="ar"/>
+		<property name="uma_make_cmd_as" value="xlC_r"/>
+		<property name="uma_make_cmd_cc" value="xlC_r"/>
+		<property name="uma_make_cmd_cpp" value="cpp"/>
+		<property name="uma_make_cmd_cxx" value="$(CC)"/>
+		<property name="uma_make_cmd_cxx_dll_ld" value="xlc_r"/>
+		<property name="uma_make_cmd_cxx_exe_ld" value="$(CC)"/>
+		<property name="uma_make_cmd_dll_ld" value="xlc_r"/>
+		<property name="uma_make_cmd_exe_ld" value="xlc_r"/>
+		<property name="uma_make_cmd_ppc_gcc_cxx" value="gcc"/>
+		<property name="uma_make_cmd_ranlib" value="ranlib"/>
+		<property name="uma_processor" value="ppc,ppc64"/>
+		<property name="uma_type" value="unix,linux"/>
+	</properties>
+	<features>
+		<feature id="cmprssptrs"/>
+		<feature id="combogc"/>
+		<feature id="core"/>
+		<feature id="dbgext"/>
+		<feature id="gpu"/>
+		<feature id="se"/>
+		<feature id="se60_26"/>
+		<feature id="se7"/>
+		<feature id="se70_27"/>
```

Comment:
are all of these features still valid/used?  
why se 6.26, 7, 7.27 and not 8/9/10?